PowerEditor/src/WinControls/Preference/preferenceDlg.h

```diff
@@ -52,8 +52,16 @@ private :
 
 class ToolbarSubDlg : public StaticDialog
 {
+friend class PreferenceDlg;
 public:
 	ToolbarSubDlg() = default;
+	~ToolbarSubDlg() {
+		if (_accentTip)
+		{
+			::DestroyWindow(_accentTip);

```

Comment:
Already done:
https://github.com/notepad-plus-plus/notepad-plus-plus/blob/master/PowerEditor/src/WinControls/Preference/preferenceDlg.cpp#L928

Reply:
ok, but it won't be consistent with other subdialogs